interfaces/automated-testing/scd/scd.yaml

```diff
@@ -35,7 +35,478 @@ components:
+          type: array
+          items:
+            $ref: '#/components/schemas/Volume4D'
+
+    FlightAuthorisationData:
+      description: |-
+        A dataset to hold details of a UAS flight authorization request matching some of the requirements of Annex IV of the U-Space regulation 2021/664
+      type: object
+      properties:
+        uas_serial_number:
+          type: string
+          example: "T4S6BBMSZV4H0650"
+          description: Serial number of the UAS per the CTA-2063-A standard
+        operation_mode:   
+          $ref: '#/components/schemas/OperationMode'
+        operation_category:
+          type: string
+          example: "normal"
+          description: |-
+            Specify if the flight is "special" or has a specific priority.
+        uas_class: 
+          $ref: '#/components/schemas/UASClass'
+        identification_technologies: 
+          type: array
+          items:
+            type: string
+            enum:
+              - Undeclared
+              - Network
+              - Broadcast
+            description: Specify an identification technology that will be used for this flight
+        connectivity_methods:
+          type: array
+          items:
+            type: string
+          example: ['radio']
+        endurance_minutes: 
+          type: number
+          description: A number to specify the endurance of the drone in minutes
+        emergency_procedure_url:
+          type: string
+          format: url
+          description: A valid url to contact in case of emergencies
+        operator_id:
+          type: string
+          example: "CHE0n4fujx7nwanb-fwi"
+          description: ID assigned to the operator per the EN4709-02 standard
+      required:
+      - uas_serial_number
+      - operation_mode
+      - uas_class
+      - identification_technologies
+      - operator_id
+
+    OperationMode:
+      type: string
+      enum:
+        - Undeclared
+        - Vlos
+        - Bvlos
+      description: Specify if the operation is a `vlos` or `bvlos` operation
+
+    UASClass:
+      type: string
+      enum:
+        - C0
+        - C1
+        - C2
+        - C3
+        - C4
+      description: Specify the class of the UAS to be flown, the specifition matches EASA class identification label categories, for more information see EASA website
+
+    InjectFlightRequest:
+      type: object      
+      properties: 
+        injection_id: 
+          description: 'ID of this injection.  Remains the same regardless of the flight ID reported in the system.'
+          type: string
+          example: 'edb7695f-8737-4b9f-91f8-e2afbb333f41'
+
+        operational_intent:
+          description: A Operational Intent for a flight provided for the test
+          $ref: '#/components/schemas/OperationalIntentTestInjection'
 
+        flight_authorisation:
+          description: A set of data about the operator to authorize a flight
```

Comment:
May I suggest to update the description like this:
```
Full description of a flight authorisation including mandatory information required by ANNEX IV of COMMISSION
        IMPLEMENTING REGULATION (EU) 2021/664 for an UAS flight authorisation request.
        Reference: https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32021R0664&from=EN#d1e32-178-1
```

Reply:
Thanks I have now added this to the YAML

Reply:
I have done this in the definition now.